algorithms/kernel/kernel.h

```diff
@@ -175,7 +175,7 @@
                             DAAL_KERNEL_AVX512_CONTAINER(ContainerTemplate, __VA_ARGS__)>::ClassName(daal::services::Environment::env * daalEnv)    \
         : BaseClassName(daalEnv), _cntr(NULL)                                                                                                       \
     {                                                                                                                                               \
-        switch (daalEnv->cpuid)                                                                                                                     \
+        GetCpuid switch (daalEnv->cpuid)                                                                                                            \
```

Comment:
Please change "daalEnv->cpuid" to "cpuid"

Reply:
Thanks!